srcpkgs/ergo/files/ergo/log/run

```diff
@@ -0,0 +1 @@
+/usr/bin/vlogger
```

Comment:
please `exec`, like all other `log/run`

Reply:
It's just a symlink to `vlogger`, as I saw some other packages do, is this undesirable now?

Reply:
sorry i wasn't aware. also see https://github.com/void-linux/void-packages/pull/42026

Reply:
Fair enough, but concretely what do I go for then? Symlink or separate `vlogger` call?

Reply:
rebase and you shouldn't need to do either :)

Reply:
I noticed something was done about more consistent logging, will do asap